### PR TITLE
dev-db/percona-xtrabackup, bug #606110

### PR DESCRIPTION
**Pull request by manwe-pl**
dev-db/percona-xtrabackup stable for x86 and amd64, masking system-boost use flag, cleanup of older version

### CODE REVIEW

File: dev-db/percona-xtrabackup/percona-xtrabackup-2.4.5.ebuild [hunk 1]
```diff
@@ -11,7 +11,7 @@ SRC_URI="https://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-${PV}/s
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="~amd64 ~x86"
+KEYWORDS="amd64 x86"
```
Comment (SoapGentoo) [inline dev-db/percona-xtrabackup/percona-xtrabackup-2.4.5.ebuild]: you're stabilising an ebuild, this isn't allowed by users in general. Please speak to the maintainer such that he opens a STABLEREQ bug for it.

Comment (manwe-pl) [inline dev-db/percona-xtrabackup/percona-xtrabackup-2.4.5.ebuild]: I am one of the maintainers https://github.com/gentoo/gentoo/blob/master/dev-db/percona-xtrabackup/metadata.xml#L9 

And we do have STABLEREQ bug: https://bugs.gentoo.org/show_bug.cgi?id=606110

Comment (gktrk) [inline dev-db/percona-xtrabackup/percona-xtrabackup-2.4.5.ebuild]: you don't need a pull request for this, the members of the corresponding arch teams will test the package and stabilize it for you

Comment (manwe-pl) [inline dev-db/percona-xtrabackup/percona-xtrabackup-2.4.5.ebuild]: Removed

**Comment by mgorny** [comment]
The relevant bug seems to be fixed, and there's no activity in here.

**Pull request closed without merging**